src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -13958,42 +13958,24 @@ void VRFCondenserEquipment::VRFOU_CalcCompC(EnergyPlusData &state,
                 };
 
                 General::SolveRoot(state, 1.0e-3, MaxIter, SolFla, SmallLoadTe, f, MinOutdoorUnitTe,
-                                   T_suction); // SmallLoadTe is the updated Te'
-                if (SolFla == -1) {
-                    // show error not converging
-                    ShowWarningMessage(state, format("{}: low load Te adjustment failed for {}", RoutineName, this->Name));
-                    ShowContinueErrorTimeStamp(state, "");
-                    ShowContinueError(state, format("  Iteration limit [{}] exceeded in calculating OU evaporating temperature", MaxIter));
-                } else if (SolFla == -2) {
-                    this->LowLoadTeError++;
-                    if (LowLoadTeError < 5) {
-                        ShowWarningMessage(state,
-                                           format("{}: no Te solution was found for {} f({})={} and f({})={} are the same sign",
-                                                  RoutineName,
-                                                  this->Name,
-                                                  MinOutdoorUnitTe,
-                                                  f(MinOutdoorUnitTe),
-                                                  T_suction,
-                                                  f(T_suction)));
-                        ShowContinueErrorTimeStamp(state, "");
-                    }
-                    ShowRecurringWarningErrorAtEnd(state,
-                                                   "Low load calculation Te solution not found as end points have the same sign",
-                                                   this->LowLoadTeErrorIndex,
-                                                   SolFla,
-                                                   SolFla);
-                    if (f(T_suction) < 0) {
-                        // demand < capacity at both endpoints of the Te range, assuming f(x) is roughly monotonic than this is the low load case
-                        // TeTol is added to prevent the final updated Te to go out of bounds
-                        SmallLoadTe = MinOutdoorUnitTe + TeTol; // MinOutdoorUnitTe; //SmallLoadTe( Te'_new ) is constant during iterations
-                    } else {
-                        // demand > capacity at both endpoints of the Te range, take the end point x where f(x) is closer to zero
-                        if (f(MinOutdoorUnitTe) > f(T_suction)) { // f(T_suction > 0, not equal as SolFla will not be -2
-                            SmallLoadTe = T_suction;
-                        } else {
-                            SmallLoadTe = MinOutdoorUnitTe;
-                        }
+                                   T_suction + 6); // SmallLoadTe is the updated Te'
```

Comment:
1) Why is this SolveRoot call, lines 13960 and 13961, still here? It doesn't do anything.
2) If you are going to execute line 13968 anyway, then you may as well make this an `if () { } else { }` to eliminate the extra equivalence.
3) I would still like to see a warning for SolFla == -2 and let the code coverage tell us it's never used than to possibly miss a coding mistake. At the very least put an assert there so a developer might stumble across an issue.
4) What happened to the recurring warning?

Reply:
@rraustad Thanks for the comments.
for 1. I think might have forgot to change it back. Now the upper bound is changed back to `T_suction`.
2. I changed it to if else logic
3. I added back the warnings for the SolFla = -2 case
4. Also added back recurring warnings in the SolFla = -2